fronts-client/src/components/Home.tsx

```diff
@@ -52,6 +55,16 @@ const Home = ({ availableEditions }: IProps) => (
 
 const mapStateToProps = (state: State) => ({
   availableEditions: selectAvailableEditions(state),
+  editEditionsIsPermitted: selectEditionsPermission(state)?.['edit-editions'],
 });
 
+const displayNoPermissionMessage = (onContent: string) => {
+  return (
+    <p>
+      You do not have permission to edit {onContent}. Please contact
+      central.production@guardian.co.uk to request access.
```

Comment:
Non-blocking:

```suggestion
      <a href="mailto:central.production@guardian.co.uk">Central Production</a> to request access.
```

(or similar)